### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by cesss** [stella-emu/stella#1002] [Q&A] [answered]
Are you very far from doing a new release? I'd like to be able to play Indy 500 with the driving controllers connected to 2600-daptors, which is broken in 6.7, and fixed in the source, but there wasn't a release since the fix. I'm currently using two Stellas: 6.7 and 5.1.3 just because of that bug.

**Comment by thrust26** [comment]
There might be a new release in 2024.

**Reply by SRC267** [reply, marked as answer]
v7.0 is now out
